unity-environment/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.1.0b6
+m_EditorVersion: 2018.1.0b9
```

Comment:
Would it make sense to ignore this file from git?

Reply:
It is actually needed for Unity when importing a project into a new version of the Editor.